lib/database/index.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+const { Database } = require('../application/interfaces');
+
+/**
+ * Memory implementation of the Database interface.
+ */
+class MemoryDatabase extends Database {
+    /**
+     * Returns all available repositories.
+     */
+    get repositories() {
+        return require('./repositories');
+    }
+
+    /**
+     * Performs connection to the database.
+     *
+     * @returns {Promise} Promise object represents the outcome.
+     */
+    async connect() {
+        // This is a no-op method
+    }
```

Comment:
Are empty bodies needed? this looks cleaner?
```suggestion
    async connect(); // This is a no-op method
```

Reply:
I would go for the current one with brackets as when the implementation changes it will only show changes in the body of the method instead of the whole function.